b2sdk/raw_api.py

```diff
@@ -435,23 +437,27 @@ class B2RawHTTPApi(AbstractRawApi):
     def __init__(self, b2_http):
         self.b2_http = b2_http
 
-    def _post_json(self, base_url, api_name, auth, **params) -> dict[str, Any]:
+    def _post_json(self, base_url: str, endpoint: str, auth: str, **params) -> JSON:
```

Comment:
You're extending return type, it used to be `dict`, now it may be even an `int`. So... Is there any endpoint returning not `dict`?

Reply:
currently, I have yet to see an B2 endpoint that doesn't return `dict` on top level.
It is however quite common to have `list` as top-level object.

The main improvement here is that this type annotation is recursive instead of "dict[str, Any]"